pandas_gbq/gbq.py

```diff
@@ -604,6 +604,7 @@ def _download_results(
     def load_data(
         self,
         dataframe,
+        project_id_table,
```

Comment:
I don't have that strong a view, but I'm not sure if using `project_id_table` is clearer or than just `project_id`

Reply:
Many thanks for this sensible comment. I thought that this variable name would make things clearer, because we refer here to the project in which the destination table is located, and not the one associated to the BQ client. However, if you prefer to use another variable name, I would be open for that. Maybe @tswast has an opinion on this? Best regards.